### ISSUE TITLE
Issue with locomotives spawned with God Mode 0.2.0 + some other issue

### ISSUE THREAD

**Issue by myron-obsolete-account** [FoobianBlock/TSW-ModdersLog#1]
Hi um I don't know where else to put this but can you maybe help me with a specific issue with the God Mode mod?
When I spawn the following BR's: 182, 185 and 146.2, they refuse to start driving despite all brakes being released, reverser set to on etc.

Any way to get that fixed? The locos do seem to work when spawned automatically when loading a map (e.g. the BR 182 does drive when going into timetable mode or just starting a scenario, and the 185 thats pre-spawned in the Training Center also moves fine, so maybe the way of the locos spawning could be changed or idk, I'm not a professional coder. It appears to only happen to the locomotives because the electrical multiple units BR 403 and 401 ICE's 1 and 3's do work.

---

Oh and btw. when I'm already talking about issues here, sometimes when spawning the like second or third train on the tracks, all switches are set somewhere random e.g. the ones in front of the driver's seat are all facing to the driver. And some other switches set to somewhere aren't being recognized at first. For example on one loco the battery was set to off but since there was no actual interaction with it the game still thought it was on. As soon as I touched the switch the power just disappeared because the game now realized that the battery switch was turned off. This all may sound so cringe but could you please get that fixed? Would be so kind, thank you.
God Mode is an amazing mod and thank you for that.

**Comment by FoobianBlock** [comment]
The locos you mentioning having a traction lock after spawning is a known nuicanse which has already been discussed on the comment section for God Mode on Train Sim Community, but I don't blame you for not scrolling through all of that. The simple fix is cycling the AFB once. Don't think I can do anything about that and will blame DTG for that.

The other issue(s) you mentioned may be save game related? I remember having seen something like this a while ago now, but wasn't sure what it was about or what caused it. Maybe you can help me out with that, even if it's just you recording a video which includes you spawning the trains and me trying to reproduce it.

**Comment by MoRoN222** [comment]
Thank you for the AFB tip. Really sad and annoying that it's like that. And I actually did scroll through the comment section (but not because of the issue) and I probably just overlooked it.

And the other issue is, yes, actually game-saving related. I have no idea how and why it happens but ehh. Gotta stop saving games then. And I did make a [video](https://youtu.be/utQE7FWdYmQ) even though the problem is clear now, but lets just say it's for scientific reasons. 🍷🗿

**Comment by MoRoN222** [comment]
btw just as an idea have you ever thought about making it possible to save formations to the formations list (or a new section called custom formations) and then people can give them their own names? a good example is that the BR 187 doesnt have dosto formations, however you can couple the loco to dosto coaches. wouldnt it be cool to save that formation and then use it later or on a different route? I also made a very VERY long ICE 1 with about 43 coaches and I'd like to reuse it sometime soon too. ain't that an awesome idea duh?

**Comment by MoRoN222** [comment]
How the hell do I make a soundmod tho

**Comment by FoobianBlock** [comment]
> have you ever thought about making it possible to save formations to the formations list (or a new section called custom formations)

Yes I have actually! I haven't tried out my concept yet, but want to look at that in the near future

**Comment by MoRoN222** [comment]
> Yes I have actually! I haven't tried out my concept yet, but want to look at that in the near future

Oh, that sounds cool!

**Comment by MoRoN222** [comment]
hey @FoobianBlock could you maybe help me with soundmodding because I'm a noob in things like that and I only want to change some sounds for the ICE 3, I want to change the horn of it into the TGV horn because that one sounds better for the ICE 3 in my opinion
